Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>print(\"Hy\" + \"Kā iet?\")\r\nprint(\"Hy \" + \"Kā iet?\")\r\nprint(\"Hy\" + \" \" + \"Kā iet?\")\r\n\r\nkad ievievieto vērtības\r\n\r\nprint(\"Tevi sauc (v1)\" + input(\"Vārds ir\"))\r\n\r\nvards = input(\"Kā tevi sauc\\n\")\r\nprint(\"Tavs vārds ir\" + vards)\r\n\r\nnum1 = 1\r\nnum2 = 2\r\n\r\nprint(num1 + num2)\r\nrezultats = int(num1 + num2)\r\nprint(rezultats)\r\n\r\ninpu = \"Biete's\"\r\nprint(len(inpu))\r\n\r\nname = input(\"Kā tevi sauc? \\n\")\r\n\r\nprint(\"Tavas grupas nosaukums ir \" + name + \" grupa\" )
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
--- a/main.py	(revision 6c339ffc7367814f64cfd721d99163ed35eb32f2)
+++ b/main.py	(date 1719240301937)
@@ -2,7 +2,6 @@
 print("Hy " + "Kā iet?")
 print("Hy" + " " + "Kā iet?")
 
-kad ievievieto vērtības
 
 print("Tevi sauc (v1)" + input("Vārds ir"))
 
